src/data/finset/finsupp.lean

```diff
@@ -0,0 +1,86 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import data.finsupp.basic
+
+/-!
+# Finitely supported produvy of finsets
+
+This file defines the finitely supported product of finsets as `finset (ι →₀ α)`.
+
+## Main declarations
+
+* `finset.finsupp`: The finitely supported product of finsets.
+* `finsupp.pi`: `f.pi` is the finset of `finsupp`s whose `i`-th value lies in `f i`.
```

Comment:
Would be good to add a quick note here about the difference between these two notions, as they are quite similar at first glance

Reply:
What about now?

Reply:
yeah perfect!